content/en/docs/reference/kube-scheduler/profiles.md

```diff
@@ -0,0 +1,174 @@
+  Extension points: *Score*.
+- `NodeAffinity`: Implements
+  [node selectors](/docs/concepts/configuration/assign-pod-node/#nodeselector)
+  and [node affinity](/docs/concepts/configuration/assign-pod-node/#node-affinity).
+  Extension points: *Filter*, *Score*.
+- `PodTopologySpread`: Implements
+  [Pod topology spread](/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
+  Extension points: *PreFilter*, *Filter*, *PreScore*, *Score*.
+- `NodeUnschedulable`: Filters out nodes that have `.spec.unschedulable` set to
+  true.
+  Extension points: *Filter*.
+- `NodeResourcesFit`: Checks if the node has all the resources that the Pod is
+  requesting.
+  Extension points: *PreFilter*, *Filter*.
+- `NodeResourcesBallancedAllocation`: Favors nodes that would obtain a more
+  balanced resource usage if the Pod is scheduled there.
+  Extension points: *Score*.
+- `NodeResourcesLeastAllocated`: Favors nodes that have a low allocation of
+  resources.
+  Extension points: *Score*.
+- `VolumeBinding`: Checks if the node has or if it can bind the requested
+  volumes.
+  Extension points: *Filter*.
+- `VolumeRestrictions`: Checks that volumes mounted in the node satisfy
+  restrictions which are specific to the volume provider.
+  Extension points: *Filter*.
+- `VolumeZone`: Checks that volumes requested satisfy any zone requirements they
+  might have.
+  Extension points: *Filter*.
+- `NodeVolumeLimits`: Checks that CSI volume limits can be satisfied for the
+  node.
+  Extension points: *Filter*.
+- `EBSLimits`: Checks that EBS volume limits can be satisfied for the node.
+  Extension points: *Filter*.
+- `GCEPDLimits`: Checks that GCP-PD volume limits can be satisfied for the node.
+  Extension points: *Filter*.
+- `AzureDiskLimits`: Checks that Azure disk volume limits can be satisfied for
+  the node.
+  Extension points: *Filter*.
+- `InterPodAffinity`: Implements
+  [inter Pod affininity and antiaffinity](/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
+  Extension points: *PreFilter*, *Filter*, *PreScore*, *Score*.
+- `PrioritySort`: Provides the default priority based sorting.
+  Extension points: *QueueSort*.
+- `DefaultBinder`: Provides the default binding mechanism.
+  Extension points: *Bind*.
+  
+You can also enable the following plugins, through the component config APIs,
+that are not enabled by default:
+
+- `NodeResourcesMostAllocated`: Favors nodes that have a high allocation of
+  resources.
+  Extension points: *Score*.
+- `RequestedToCapacityRatio`: Favor nodes according to a configured function of
+  the allocated resources.
+  Extension points: *Score*.
+- `NodeResourceLimits`: Favors nodes that satisfy the Pod resource limits.
+  Extension points: *PreScore*, *Score*.
+- `CinderVolume`: Checks that Cinder volume limits can be satisfied for the
+  node.
+  Extension points: *Filter*.
+- `NodeLabel`: Filters and/or scores a node according to the configured labels.
```

Comment:
```suggestion
- `NodeLabel`: Filters and / or scores a node according to configured
  {{< glossary_tooltip text="label(s)" term_id="label" >}}.
```
(do you configure labels or selectors? There's a glossary entry for selector as well.)

Reply:
Labels